modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -1,26 +1,76 @@
+
+== How Application Deployments Work in Runtime Fabric on Self-Managed Kubernetes 
+
+When you deploy an application in Runtime Fabric on Self-Managed Kubernetes, the following occurs:
+
+. You use Runtime Manager to trigger the application deployment.
+. The Runtime Fabric agent in the target cluster receives a request to deploy the application.
+. Runtime Fabric assigns the application's namespace or creates a new namespace if necessary.
+. Runtime Fabric generates the appropriate Kubernetes resources, including a deployment, ConfigMap, secrets, services, and ingress.
+. The Kubernetes deployment resource may include an init container, which fetches dependencies from the Runtime Fabric resource cache.
+. If the resource cache doesn’t contain required dependencies, Runtime Fabric fetches them from the control plane and adds them to the resource cache.
+
+//image 
+
+=== Assigning Namespaces in Runtime Fabric
+
+Each application is deployed into a Kubernetes namespace based on the application’s environment. 
+
+. Runtime Fabric looks for a namespace with the label `rtf.mulesoft.com/envId=<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that label, it looks for a namespace with the name `<ANYPOINT_ENVIRONMENT_ID>`. 
+. If Runtime Fabric can't find that namespace, it creates a new namespace called `<ANYPOINT_ENVIRONMENT_ID>`.
+
+//add image
+
+=== Monitoring Application Deployments
+
+The Runtime Fabric agent monitors Kubernetes Deployments labelled with `rtf.mulesoft.com/id`. When Kubernetes updates the state of the Deployment, the agent sends that update to the control plane. See xref:logs[Logs] and xref:monitoring[Monitoring] for addtional information. 
+
+
+== Runtime Fabric on Self-Managed Kubernetes: A Shared Responsibility
+
+The successful operation of Anypoint Runtime Fabric on Self-Managed Kubernetes is a shared responsibility. It is critical to understand which areas you provide and manage and which areas MuleSoft provides.
 
 The following image illustrates different MuleSoft and customer responsibilities for on-premises Runtime Fabric instances:
 
+//update image
+
 image::runtimefabricresponsibility-self-managed.png[Runtime Fabric on Self-Managed Kubernetes Shared Responsiblities]
 
-* MuleSoft responsibility:
+* MuleSoft provides:
 +
-MuleSoft is responsible for the Runtime Fabric agent, Mule runtime engine, and other dependencies used to deploy Mule applications. The Runtime Fabric agent deploys and manages Mule applications by generating and updating Kubernetes resources such as deployments, pods, replicasets, and default ingress resources.
+MuleSoft provides the Runtime Fabric agent, Mule runtime engine, and other dependencies used to deploy Mule applications. The Runtime Fabric agent deploys and manages Mule applications by generating and updating Kubernetes resources such as deployments, pods, replicasets, and ingress resources.
```

Comment:
I don't see "Runtime Fabric agent" in the product naming list; is this really Anypoint Runtime Manager agent?

Reply:
```suggestion
MuleSoft provides the Anypoint Runtime Manager agent, Mule runtime engine (Mule), and dependencies used to deploy Mule applications. The agent deploys and manages Mule applications by generating and updating Kubernetes resources such as deployments, pods, ReplicaSets, and ingress resources.
```

Reply:
No, the Runtime Fabric agent is just a service that runs in an RTF cluster and communicates with the control plane.